server/game/core/Constants.ts

```diff
@@ -7,7 +7,6 @@ export enum Location {
     Discard = 'discard',
     GroundArena = 'ground arena',
     Hand = 'hand',
-    Leader = 'leader',
```

Comment:
What is the reason for removing a dedicated location for Leaders?

Reply:
Yeah so this was actually a mistake on my part that I was slow to correct. Per the swu rules, leader and base are actually both in the same "base zone" and there is no leader zone.